src/index.js

```diff
@@ -74,6 +76,10 @@ const processMode = mode => {
     if (mode.hideSelectTrainer) {
       store.dispatch(setSelectedTrainer(mode.hideSelectTrainer));
     }
+
+    if (mode.randomizeTestData) {
```

Comment:
Do we think we'll want to offer more than just random at some point?

Reply:
Potentially in the far future, but not in the current unit.

Reply:
My one thought was whether we wanted to expose this parameter a bit more generally, but if you're comfortable only exposing randomization, then that seems fine.

Reply:
Yeah, I thought about that too and feel like we can add that if we need it or roll it into levelbuilder improvements; I suspect form fields rather than a JSON blob of parameters will be a nicer long-term interface once the parameters are completely solidified.